code/src/Components/Summary.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Summary = ({ userYearofBirth, userGender, userActivities, userExpectation, userRecommendation, userTextInput }) => {
+
+    return (
+        <div>
+            <p>
+                Jag är en {userGender} född år {userYearofBirth}. 
+                Jag har deltagit i aktiviteten {userActivities}. 
+                {userActivities} kan jag {userRecommendation} rekommendera
+                till en vän. Aktiviteten uppfyllde {userExpectation} mina förväntningar. 
+                Jag vill också skicka med er {userTextInput}.
+            </p>
```

Comment:
Recommendation is to change them to eg. several <p> tags so it gets space in the summary at the webpage. Just to read the summary easier :)